### DISCUSSION TITLE
NPM Helper scripts not fully installing?

### DISCUSSION THREAD

**Discussion by paulgali** [tteck/Proxmox#3005] [Help Needed] [answered]
Ran [this ](https://helper-scripts.com/scripts?id=43dhsvufclq5a17)on node the shell to set up the container. Set up fine, but hitting the ip (at port 81) didn't result in anything. 

Hitting the IP alone, shows openresty; so something is definitely working but NPM itself is now missing from installs.

![image](https://github.com/tteck/Proxmox/assets/23615789/c61925ea-b1b2-437b-9e3b-da9af9843e88)

Running the bash in the LXC shell tells me nothing is installed. Is there an issue with this current script?

![image](https://github.com/tteck/Proxmox/assets/23615789/c90a48fd-461f-457d-aba6-9b1e9da85def)

**Comment by tteck** [comment, marked as answer]
❗ It is crucial not to exit the Proxmox shell while running scripts. Leaving the shell during script execution will result in the script being interrupted.

**Reply by paulgali** [reply]
3rd times the charm i guess. I will delete this thread - I'm sure i had it open, sorry! (Nice update on the website btw)

**Reply by Mellowlynx** [reply]
TIP: Open the shell in a pop-up, so you don't close it accidentally when navigating away😄.

**Reply by paulgali** [reply]
Yeah I did that. I think im having a network issue on top of this. This is a strange error...

![image](https://github.com/tteck/Proxmox/assets/23615789/61f97556-6fea-4546-887c-b4d83e1d0c02)

**Reply by tteck** [reply]
If the default Linux distribution is not adhered to, script support will be discontinued.
